python/lsst/obs/comCam/ingest.py

```diff
@@ -18,14 +18,30 @@ def __init__(self, config, *args, **kwargs):
         super(ParseTask, self).__init__(config, *args, **kwargs)
 
     def getInfo(self, filename):
-        # Grab the basename
+        """ Get the basename and other data which is only available from the filename/path.
+        
+        This seems fragile, but this is how the teststand data will *always* be written out, 
+        as the software has been "frozen" as they are now in production mode.
+
+        Parameters
+        ----------
+        filename : `str`
+            The filename
+
+        Returns
+        -------
+        phuInfo : `dict`
```

Comment:
I'm happy to change this, but this was a conscious decision to adhere to the same naming scheme as the base class, and to be the same as other ingest.getInfo() methods in other obs_packages. Would you really like me to change this if it means no longer having that symmetry?